.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
+- [ ] (Feedback from reviewers)
+
+## How To Test
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
```

Comment:
Right... linters... that's a good point. I was just thinking strictly about code-style, but we have "config" in this repo as well.